src/sentry/api/endpoints/sentry_app_installation_external_issue_details.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.response import Response
+
+from sentry.api.bases import (
+    SentryAppInstallationExternalIssueBaseEndpoint as ExternalIssueBaseEndpoint,
+)
+from sentry.mediators import external_issues
+from sentry.models import PlatformExternalIssue
+
+
+class SentryAppInstallationExternalIssueDetailsEndpoint(ExternalIssueBaseEndpoint):
+    def delete(self, request, installation, external_issue_id):
+        try:
+            platform_external_issue = PlatformExternalIssue.objects.get(
+                id=external_issue_id,
+                project__organization=installation.organization,
+                service_type=installation.sentry_app.slug,
```

Comment:
@ceorourke @MeredithAnya Just want to point out this could potentially cause problems if the slug of a Sentry App ever changes. We rarely make that modification but it's worth knowing what would break if we do that.